pypeit/core/wavecal/wv_fitting.py

```diff
@@ -37,6 +37,7 @@ class WaveFit(datamodel.DataContainer):
                  'wave_fit': dict(otype=np.ndarray, atype=np.floating,
                                   descr='Wavelength IDs assigned'),
                  'xnorm': dict(otype=float, descr='Normalization for fit'),
+                 'fwhm': dict(otype=float, descr='Estimate FWHM of arc lines in pixels'),
```

Comment:
As above, we should specify here whether it is native pixels and whether it is the Arc's pixels or the science  (as their binning can be different).

Reply:
Done